Integrations/integration-EWSMailSender.yml

```diff
@@ -197,17 +210,36 @@ script:
 
 
     def test_module():
+        BaseProtocol.TIMEOUT = 15
```

Comment:
In what scale is this timeout?

Reply:
It's only when test (for authentication only). I think it's enough. I can increase it if you think it's too low